### ISSUE TITLE
Exposing strokeWidth as a property

### ISSUE THREAD

**Issue by tsejerome** [ianmiller347/feather-icons-react#43]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `feather-icons-react@0.6.2` for the project I'm working on. I loved that we can expose the package and just use the `icon` parameter to decide which icon to use instead of importing svg elements one by one.

Anyway, I'm trying to adjust the strokeWidth with this package but I realize that it is not exposed as one of the parameter. I believe this can be easily done via changing two lines of code.

**How it will look like in code:**
<img width="433" alt="image" src="https://github.com/ianmiller347/feather-icons-react/assets/20391660/2e15ff0e-0833-4f55-b221-f5d2ca5bb4e6">

**Result:**
<img width="212" alt="image" src="https://github.com/ianmiller347/feather-icons-react/assets/20391660/adfa3f09-0076-4ffe-9827-83f997311594">

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/feather-icons-react/build/FeatherIcon.js b/node_modules/feather-icons-react/build/FeatherIcon.js
index 0d184cf..87dd873 100644
--- a/node_modules/feather-icons-react/build/FeatherIcon.js
+++ b/node_modules/feather-icons-react/build/FeatherIcon.js
@@ -84,6 +84,7 @@ var FeatherIcon = function FeatherIcon(_ref) {
   var icon = _ref.icon,
       _ref$size = _ref.size,
       size = _ref$size === void 0 ? 24 : _ref$size,
+      strokeWidth = _ref.strokeWidth || "2",
       _ref$className = _ref.className,
       className = _ref$className === void 0 ? '' : _ref$className,
       _ref$fill = _ref.fill,
@@ -100,7 +101,7 @@ var FeatherIcon = function FeatherIcon(_ref) {
     viewBox: "0 0 24 24",
     fill: fill,
     stroke: "currentColor",
-    strokeWidth: "2",
+    strokeWidth: strokeWidth,
     strokeLinecap: "round",
     strokeLinejoin: "round",
     className: "feather feather-".concat(icon, " ").concat(className)
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by ianmiller347** [comment]
Thanks for raising the issue @tsejerome 

More than happy to accept a pull request for this, as i believe it's a good idea to expose this prop. 

I may have time this weekend to make it myself, but since you already have the change ready, you can definitely put up a PR and i'll review it and test it then we can include it in the next version and publish it

**Comment by ianmiller347** [comment]
Done. published with version 0.7.0

Resolving.

**Comment by tsejerome** [comment]
Sorry buddy I totally missed the message but thank you for the awesome package!
